spec/support/models/api/v4/run.json

```diff
@@ -97,6 +130,17 @@
           }
         }
       }
+    },
+    "segments": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "attempt_number",
+          "realtime_duration_ms",
+          "gametime_duration_ms"
+        ]
+      }
```

Comment:
As an unwritten rule I like to be as strict as possible in the schema, which means always specifying `"additionalProperties": false` for objects and always fully describing all the field types by using the `properties` key, like you did above for the root run object.

Reply:
Got it, will update the schema.